spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+RSpec.describe Category, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
install gem https://github.com/thoughtbot/shoulda-matchers and write test that tests that category belongs to user. 

` it { should belong_to(:user) }`